autoeq/constants.py

```diff
@@ -292,4 +292,38 @@
             'max_fc': 10000.0,
         }] * 8
     },
+    'VOICEMEETER_PEQ': {
+        'optimizer': {
+            'min_f': 20,
+            'max_f': 20000,
+            'max_time': None,
+            'min_change_rate': None,
+            'min_std': 0.008,
+        },
+        'filter_defaults': {
+            'min_fc': 20.0,
+            'max_fc': 10000.0,
+            'min_q': 0.18248,
+            'max_q': 6.0,
+            'min_gain': -20,
+            'max_gain': 20
+        },
+        'filters': [{
+            'type': 'LOW_SHELF',
+            'fc': 65.406392,
```

Comment:
@JuhaTP Is the center frequency for the low shelf filter fixed? If not, what are the minimum and maximum configurable values?

Reply:
Not fixed. Value there is probably for test headphones I used.